### PR TITLE
dashboard: setup message polling

### PR DESCRIPTION
**Pull request by jpcloureiro**
setup message polling when on tickets view.

add dummy_contacts for demo purposes. this means when we use any of those values for signin or signup, ownership validation gets bypassed.

fix message position & color on messaging modal based on the message sender

### CODE REVIEW

File: src/app/views/Dashboard.jsx [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+    const ticketId = ticket.id;
     onRowClick(ticketId);
   };
 
+  const updateTickets = React.useCallback(() => setTimeout(onTicketUpdate, 3000), [
+    onTicketUpdate,
+  ]);
+
+  React.useEffect(() => {
+    console.log("why");
```
Comment (consAtanasov) [inline src/app/views/Dashboard.jsx]: Why :D

File: src/app/views/App.jsx [hunk 2]
```diff
@@ -206,13 +196,15 @@ const App = () => {
...
     }
+    setError('');
     console.log(createProjectResult);
     // if all goes ok, then set the state to display the code and signIn with the provided contract
     const isSignedInSuccessfully = await signIn(contractAddress);
     if (!isSignedInSuccessfully) {
       setError(`Cannot Sign with with contract address ${contractAddress}`);
       return;
     }
+    setError('');
```
Comment (consAtanasov) [inline src/app/views/App.jsx]: 2 set errors?

File: api/tickets.js [hunk 3]
```diff
@@ -1,45 +1,57 @@
...
-    // TODO: get it from cookie, but time is short so hardcode for now
+  // TODO: get it from cookie, but time is short so hardcode for now
 
-    //const test = await getTicketsForOwner('123');
-    if (request.method.toLowerCase() === 'get') {
-        const result = await getTicketsForOwner(owner);
+  //const test = await getTicketsForOwner('123');
+  if (request.method.toLowerCase() === "get") {
+    const owner = request.query?.owner;
+    console.log(owner);
```
Comment (consAtanasov) [inline api/tickets.js]: 2 console logs. perhaps an if check if owner is null throw an error?

**Review by consAtanasov — APPROVED** [review]

File: src/app/services/db.js [hunk 4]
```diff
@@ -1,15 +1,15 @@
 const hosturl = process.env.BACKEND_HOST;
 
-export async function loadTicketsForOwner() {
-    const path = `${hosturl}/tickets`;
+export async function loadTicketsForOwner(owner) {
+    const path = `${hosturl}/tickets?owner=${owner}`;
```
Comment (naugtur) [inline src/app/services/db.js]: probably too late to do it, but in general new URL('/tickets?owner=etc', hosturl).href - you avoid issues if hosturl ended with `/`

File: src/app/views/App.jsx (hunk 2 above)
Comment (jpcloureiro) [reply src/app/views/App.jsx]: `setError('')` cleans up previous errors. e.g. when you try to sign up multiple times

File: src/app/views/Dashboard.jsx (hunk 1 above)
Comment (jpcloureiro) [reply src/app/views/Dashboard.jsx]: The struggle was real

File: api/tickets.js (hunk 3 above)
Comment (jpcloureiro) [reply api/tickets.js]: removed the duplicate log.

if check is there, on line 30-33

File: src/app/views/Signup.jsx [hunk 5]
```diff
@@ -54,7 +54,7 @@ const Signup = ({ onSubmit }) => {
               <label className="input-group">
                 <input
                   type="text"
-                  placeholder="e.g. Napoleon"
+                  placeholder="Type here"
```
Comment (martahj) [inline src/app/views/Signup.jsx:57]: The original placeholders are more helpful imo but it's not a blocker

**Review by martahj — APPROVED** [review]

**Pull request merged**